pandas/core/generic.py

```diff
@@ -7810,6 +7810,86 @@ def mask(self, cond, other=np.nan, inplace=False, axis=None, level=None,
         is not realigned. That is, use freq if you would like to extend the
         index when shifting and preserve the original data.
 
+        Examples
+        --------
+        Compute the difference between a column in a dataframe
```

Comment:
Whenever you reference it be sure to capitalize appropriately the word `DataFrame`